docs/application-flow.rst

```diff
@@ -390,17 +389,130 @@ With the run of each sector involving the following steps:
+
+With all this information, the investment process can proceed. This is done per sector using the method described by the ``lpsolver`` in the ``settings.toml`` file. Available solvers are described in :py:mod:`muse.investments`
+
+If the investment succeeds, the new installed capacity will become part of the agents' assets.
+
+
+Dispatch
+~~~~~~~~
+
+The dispatch stage when running a sector can be described by the following graph:
+
+.. graphviz::
+    :align: center
+    :alt: Dispatch stage of a sector calculation
+
+    digraph dispatch {
+            fontname="Helvetica,Arial,sans-serif"
+            node [fontname="Helvetica,Arial,sans-serif", shape=box, style=rounded]
+            edge [fontname="Helvetica,Arial,sans-serif"]
+            rankdir=LR
+            clusterrank=local
+            newrank=true
+
+            {node [shape=""]; start; end;}
+            capacity [label="Aggregate capacity\nfrom all agents"]
+            supply [label="Calculate supply"];
+            consumption [label="Calculate consumption"];
+            cost [label="Calculate cost"];
+            market [label="Create sector\nmarket"]
+            dispatch[label="dispatch_production\n(settings.toml)", fillcolor="#ffb3b3", style="rounded,filled"]
+
+
+            start ->  capacity -> supply -> consumption -> cost -> market -> end
+            dispatch -> supply
+        }
+
+After the investment stage is completed, then the new capacity of the sector is obtained by aggregating the assets of all agents of the sector. Then, the supply of commodities is calculated as requested by the ``dispatch_production`` argument defined for each sector in the ``settings.toml`` file.
+
+The typical choice used in most examples in MUSE is ``share``, where the utilization across similar assets is the same in percentage. However, there are other options available, like
+
+- ``costed``: assets are ranked by cost and the cheaper ones are allowed to service the demand first up to their maximum production.
+- ``maximum``: all the assets dispatch their maximum production, regardless of the demand.
+- ``match``: supply matches the demand within the constrains on how much an asset can produce while minimizing the overall associated costs. See :py:mod:`muse.demand_matching` for the mathematical details.
+
```

Comment:
*match* allows the choice among different metrics to rank assets, such as levelised costs and gross margin